packages/core/helper-plugin/lib/src/index.js

```diff
@@ -16,6 +16,7 @@ export { default as useAppInfos } from './hooks/useAppInfos';
 export { default as useQuery } from './hooks/useQuery';
 export { default as useLibrary } from './hooks/useLibrary';
 export { default as useCustomFields } from './hooks/useCustomFields';
+export { default as useFieldHint } from './hooks/useFieldHint';
```

Comment:
Do you see the need to export it? If possible I'd like to keep this private for now, because otherwise we have to write docs and can not change it because the community might start using it.

Reply:
ah I see, it's only exported for use in the UID field in `packages/core/admin/admin/src/content-manager/components/InputUID/index.js`

Do you think it is necessary to have a min/max description for the UID field?

Reply:
I see. If not discussed otherwise I guess it makes sense, if the user is able to select min/ max values in the CTB. In this case: could you please add a README to the hook explaining how it can be used?